src/osal/lv_sdl2.c

```diff
@@ -0,0 +1,171 @@
+/**
+ * @file lv_sdl2.c
+ *
+ */
+
+/*********************
+ *      INCLUDES
+ *********************/
+#include "lv_os.h"
+
+#if LV_USE_OS == LV_OS_SDL2
+
+#include <errno.h>
+#include "../misc/lv_log.h"
+
+/*********************
+ *      DEFINES
+ *********************/
+
+/**********************
+ *      TYPEDEFS
+ **********************/
+
+/**********************
+ *  STATIC PROTOTYPES
+ **********************/
+static int generic_callback(void * user_data);
+
+/**********************
+ *  STATIC VARIABLES
+ **********************/
+
+/**********************
+ *      MACROS
+ **********************/
+
+/**********************
+ *   GLOBAL FUNCTIONS
+ **********************/
+
+lv_result_t lv_thread_init(lv_thread_t * thread, lv_thread_prio_t prio, void (*callback)(void *), size_t stack_size,
+                           void * user_data)
+{
+    LV_UNUSED(prio);
+    thread->callback = callback;
+    thread->user_data = user_data;
+    thread->thread = SDL_CreateThreadWithStackSize(generic_callback, "LVGL", stack_size, thread);
+    return LV_RESULT_OK;
+}
+
+lv_result_t lv_thread_delete(lv_thread_t * thread)
+{
+    int ret;
+    SDL_WaitThread(thread->thread, &ret);
+    if(ret != 0) {
+        LV_LOG_WARN("Error: %d", ret);
```

Comment:
I think it is better ot use LOG_ERR instead.

Reply:
noticed